Generator/OdmGenerator.php

```diff
@@ -72,7 +72,14 @@ protected function generateMappingDocumentFiles(OutputInterface $output, BundleM
                 $output->writeln(sprintf('   ~ <info>%s</info>', $fileName));
             } else {
                 $output->writeln(sprintf('   + <info>%s</info>', $fileName));
-                copy($src_file, $dest_file);
+
+                $mappingEntityTemplate = file_get_contents($src_file);
```

Comment:
I think using this commit is fine, but these things should be fixed with commits on top in this PR